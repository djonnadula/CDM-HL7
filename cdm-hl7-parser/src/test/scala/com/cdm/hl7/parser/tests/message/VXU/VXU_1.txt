MSH|^~\&|145|AKAR|1013|UT0000|20170517010917|GPI8476|VXU^V04|EPIC_AKAR_VXU_123867|P|2.3.1||Epic_Mountain_PRD|||||||
PID|1||1001437157^^^EPIC^MRN|L000238869^^^AKARP^AKARP|MCDONAL^SHABOOYA^MARITUQE||19540222|M|MCDONAL^SHABOOYA^C^|2106-3|6300 HABICHT CT^^ANCHORAGE^AK^99504^USA^P||(999)999-9999^P^H~(999)999-2678^P^M|(907)999-9999^P^W|EN|M|PRO||999-99-9999|||N||||||||N
PV1||R|^^^AKAR||||||||||||||
ORC|OK|49466240^EPC|1787589||||^^^20170516231500^^||20170516230200|LNO8939^CHAN^JACK^^||LNO8939^CHAN^JACK^^^^^^THREEFOUR^^^^THREEFOUR~1376741256^CHAN^JACK^^^^^^NPI^^^^NPI|^^^AKAR^^^^^|(907)258-9272||||AKARHED2DS75^AKARHED2DS75^^40001001015^AKAR EMERGENCY|||||||||||I
RXA|0|999|20170516|20170516|115^TDAP^CVX|999|||00^New immunization record^NIP001~^VIS date: 2/24/2015~~|GPI8476^KEOGH^KASEY^L^|^^^AKAR^^^^^||||U5604AA||PMC|||CP|A|20170516230912|||
RXR|IM|LD
OBX|1|TS|29768-9^VIS Publication Date^LN|1|20150224||||||F|||20170516||||||||
OBX|2|TS|29769-7^VIS Presentation Date^LN|1|20170516||||||F|||20170516||||||||