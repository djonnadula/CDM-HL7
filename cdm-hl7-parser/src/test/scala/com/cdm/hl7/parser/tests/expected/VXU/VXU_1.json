{
  "0000.COMN": {
    "sending_facility": "AKAR",
    "message_control_id": "EPIC_AKAR_VXU_123867",
    "medical_record_num": "1001437157",
    "medical_record_urn": "L000238869",
    "patient_account_num": ""
  },
  "0001.MSH": {
    "field_separator": "|",
    "encoding_characters": "^~\\&",
    "sending_application": "145",
    "sending_facility": "AKAR",
    "receiving_application": "1013",
    "receiving_facility": "UT0000",
    "date_time_of_message": "20170517010917",
    "security": "GPI8476",
    "message_type": {
      "message_code": "VXU",
      "trigger_event": "V04"
    },
    "message_control_id": "EPIC_AKAR_VXU_123867",
    "processing_id": "P",
    "version_id": "2.3.1",
    "sequence_num": null,
    "continuation_pointer": "Epic_Mountain_PRD"
  },
  "0002.PID": {
    "set_id": "1",
    "patient_id": null,
    "patient_id_list": {
      "id_num": "1001437157",
      "id_check_digit": null,
      "check_digit_scheme": null,
      "assigning_authority": "EPIC",
      "id_type_code": "MRN"
    },
    "alternate_patient_id": {
      "id_num": "L000238869",
      "id_check_digit": null,
      "check_digit_scheme": null,
      "assigning_authority": "AKARP",
      "id_type_code": "AKARP"
    },
    "patient_name": {
      "family_name": "MCDONAL",
      "given_name": "SHABOOYA",
      "second_and_further_given_names_or_initials_thereof": "MARITUQE"
    },
    "mother_maiden_name": null,
    "date_time_of_birth": "19540222",
    "administrative_sex": "M",
    "patient_alias": {
      "family_name": "MCDONAL",
      "given_name": "SHABOOYA",
      "second_and_further_given_names_or_initials_thereof": "C",
      "suffix": null
    },
    "race": "2106-3",
    "patient_address": {
      "street_address": "6300 HABICHT CT",
      "other_designation": null,
      "city": "ANCHORAGE",
      "state_or_province": "AK",
      "zip_or_postal_code": "99504",
      "country": "USA",
      "address_type": "P"
    },
    "county_code": null,
    "phone_num_home": [
      {
        "telephone_num": "(999)999-9999",
        "telecommunication_use_code": "P",
        "telecommunication_equipment_type": "H"
      },
      {
        "telephone_num": "(999)999-2678",
        "telecommunication_use_code": "P",
        "telecommunication_equipment_type": "M"
      }
    ],
    "phone_num_business": {
      "telephone_num": "(907)999-9999",
      "telecommunication_use_code": "P",
      "telecommunication_equipment_type": "W"
    },
    "primary_language": "EN",
    "marital_status": "M",
    "religion": "PRO",
    "patient_account_num": null,
    "ssn_num_patient": "999-99-9999",
    "driver_license_num": null,
    "mother_name": null,
    "ethnic_group": "N",
    "birth_place": null,
    "multiple_birth_indicator": null,
    "birth_order": null,
    "citizenship": null,
    "veterans_military_status": null,
    "nationality": null,
    "patient_death_date_and_time": null,
    "patient_death_indicator": "N"
  },
  "0003.PV1": {
    "set_id": null,
    "patient_class": "R",
    "assigned_patient_location": {
      "point_of_care": null,
      "room": null,
      "bed": null,
      "facility": "AKAR"
    }
  },
  "0004.ORC": {
    "order_control": "OK",
    "placer_order_num": {
      "entity_id": "49466240",
      "namespace_id": "EPC"
    },
    "filler_order_num": "1787589",
    "placer_group_num": null,
    "order_status": null,
    "response_flag": null,
    "quantity_timing": {
      "quantity": null,
      "interval": null,
      "duration": null,
      "start_date_time": "20170516231500",
      "end_date_time": null,
      "priority": null
    },
    "parent_order": null,
    "date_time_of_transaction": "20170516230200",
    "entered_by": {
      "person_id": "LNO8939",
      "family_name": "CHAN",
      "given_name": "JACK",
      "second_and_further_given_names_or_initials_thereof": null,
      "suffix": null
    },
    "verified_by": null,
    "ordering_provider": [
      {
        "person_id": "LNO8939",
        "family_name": "CHAN",
        "given_name": "JACK",
        "second_and_further_given_names_or_initials_thereof": null,
        "suffix": null,
        "prefix": null,
        "degree": null,
        "source_table": null,
        "assigning_authority": "THREEFOUR",
        "name_type_code": null,
        "id_check_digit": null,
        "check_digit_scheme": null,
        "id_type_code": "THREEFOUR"
      },
      {
        "person_id": "1376741256",
        "family_name": "CHAN",
        "given_name": "JACK",
        "second_and_further_given_names_or_initials_thereof": null,
        "suffix": null,
        "prefix": null,
        "degree": null,
        "source_table": null,
        "assigning_authority": "NPI",
        "name_type_code": null,
        "id_check_digit": null,
        "check_digit_scheme": null,
        "id_type_code": "NPI"
      }
    ],
    "enterer_location": {
      "point_of_care": null,
      "room": null,
      "bed": null,
      "facility": "AKAR",
      "location_status": null,
      "person_location_type": null,
      "building": null,
      "floor": null,
      "location_description": null
    },
    "call_back_phone_num": "(907)258-9272",
    "order_effective_date_time": null,
    "order_control_code_reason": null,
    "entering_organization": null,
    "entering_device": {
      "id": "AKARHED2DS75",
      "text": "AKARHED2DS75",
      "name_of_coding_system": null,
      "alternate_id": "40001001015",
      "alternate_text": "AKAR EMERGENCY"
    },
    "action_by": null,
    "advanced_beneficiary_notice_code": null,
    "ordering_facility_name": null,
    "ordering_facility_address": null,
    "ordering_facility_phone_num": null,
    "ordering_provider_address": null,
    "order_status_modifier": null,
    "advanced_beneficiary_notice_override_reason": null,
    "filler_expected_availability_date_time": null,
    "confidentiality_code": null,
    "order_type": "I"
  },
  "0005.RXA": {
    "give_sub_id_counter": "0",
    "administration_sub_id_counter": "999",
    "date_time_start_of_administration": "20170516",
    "date_time_end_of_administration": "20170516",
    "administered_code": {
      "id": "115",
      "text": "TDAP",
      "name_of_coding_system": "CVX"
    },
    "administered_amt": "999",
    "administered_units": null,
    "administered_dosage_form": null,
    "administration_notes": [
      {
        "id": "00",
        "text": "New immunization record",
        "name_of_coding_system": "NIP001"
      },
      {
        "id": null,
        "text": "VIS date: 2/24/2015"
      }
    ],
    "administering_provider": {
      "person_id": "GPI8476",
      "family_name": "KEOGH",
      "given_name": "KASEY",
      "second_and_further_given_names_or_initials_thereof": "L",
      "suffix": null
    },
    "administered_at_location": {
      "point_of_care": null,
      "room": null,
      "bed": null,
      "facility": "AKAR",
      "status": null,
      "person_loc_type": null,
      "building": null,
      "loc_floor": null,
      "street_address1": null
    },
    "administered_per_time_unit": null,
    "administered_strength": null,
    "administered_strength_units": null,
    "substance_lot_num": "U5604AA",
    "substance_expiration_date": null,
    "substance_manufacturer_name": "PMC",
    "substance_treatment_refusal_reason": null,
    "indication": null,
    "completion_status": "CP",
    "action_code": "A",
    "system_entry_date_time": "20170516230912"
  },
  "0006.RXR": {
    "route": "IM",
    "administration_site": "LD"
  },
  "0007.OBX": {
    "set_id": "1",
    "value_type": "TS",
    "obsv_id": {
      "id": "29768-9",
      "text": "VIS Publication Date",
      "name_of_coding_system": "LN"
    },
    "obsv_sub_id": "1",
    "obsv_value": "20150224",
    "units": null,
    "references_range": null,
    "interpretation_codes": null,
    "probability": null,
    "nature_of_abnormal_test": null,
    "obsv_result_status": "F",
    "effective_date_of_reference_range": null,
    "user_defined_access_checks": null,
    "date_time_of_the_obsv": "20170516"
  },
  "0008.OBX": {
    "set_id": "2",
    "value_type": "TS",
    "obsv_id": {
      "id": "29769-7",
      "text": "VIS Presentation Date",
      "name_of_coding_system": "LN"
    },
    "obsv_sub_id": "1",
    "obsv_value": "20170516",
    "units": null,
    "references_range": null,
    "interpretation_codes": null,
    "probability": null,
    "nature_of_abnormal_test": null,
    "obsv_result_status": "F",
    "effective_date_of_reference_range": null,
    "user_defined_access_checks": null,
    "date_time_of_the_obsv": "20170516"
  }
}